### DISCUSSION TITLE
Neoscore 0.1.4 released with built-in keyboard and mouse handling

### DISCUSSION THREAD

**Discussion by ajyoon** [DigiScore/neoscore#58] [Announcements]
We've just released neoscore 0.1.4, the main exciting change being that we now support keyboard and mouse input handling out of the box! This should go a long way to enabling and simplifying many interactive score use-cases.

You can learn how to work with [keyboard](https://neoscore.org/api/neoscore.core.neoscore.html#neoscore.core.neoscore.set_key_event_handler) and [mouse](https://neoscore.org/api/neoscore.core.neoscore.html#neoscore.core.neoscore.set_mouse_event_handler) input at the API docs. We've also added [a new example score](https://github.com/DigiScore/neoscore/blob/main/examples/input.py) which demonstrates simple key and mouse input handlers.

**Comment by ajyoon** [comment]
@Xavman42 and @LCagmac this should simplify your use-cases. Please let us know if you find anything confusing or misbehaving!

**Reply by LCagmac** [reply]
Thank you!
